### ISSUE TITLE
Possible to change frame durations individually?

### ISSUE THREAD

**Issue by MoritzStefaner** [eugeneware/gifencoder#51]
Hi, great library, thanks!

I was wondering if it's possible to change frame durations on a per-frame-basis?

**Comment by skeddles** [comment]
yes you can just run ` encoder.setDelay` before the `encoder.addFrame` and it will change the framerate. The frame rate will stay like that unless you change it back before the next frame.

**Comment by MoritzStefaner** [comment]
Ah, great! Thanks.
